recipe/meta.yaml

```diff
@@ -7,16 +7,16 @@ package:
 
 source:
   fn: {{ name }}-{{ version }}.src.tar.gz
-  url: http://doxygen.nl/files/{{ name }}-{{ version }}.src.tar.gz
-  sha256: adcd968e10f27a15c956aaad86f544174f0e3d2d26d8d978929513ceeb09c19d
+  url: https://www.doxygen.nl/files/{{ name }}-{{ version }}.src.tar.gz
+  sha256: f352dbc3221af7012b7b00935f2dfdc9fb67a97d43287d2f6c81c50449d254e0
```

Comment:
It's so weird that Doxygen's tarballs keep changing without a version change...

Reply:
@tadeu some other weird thing is that CI was successful on this PR but failed for windows CI once merged. There is an issue with initexmf:
```
Warning: Option --admin should be specified when running this program with administrative privileges
```
As it is installed by conda package miktex (version 2.9.6753-h2fa13f4_1000 on failed and success CI), I have no clue on how to fix it.

Reply:
Thanks for noticing that, maybe @isuruf or @conda-forge/miktex can help with this?